constructor/nsis/main.nsi.tmpl

```diff
@@ -151,6 +153,10 @@ Page Custom mui_AnaCustomOptions_Show
 !define MUI_FINISHPAGE_TITLE_3LINES
 !define MUI_FINISHPAGE_TEXT __CONCLUSION_TEXT__
 #endif
+
+# Custom conclusion file(s)
+@CUSTOM_CONCLUSION_FILE@
+
 !insertmacro MUI_PAGE_FINISH
```

Comment:
Same as with the welcome page, the conclusion is offered as a replacement:

```suggestion
#if custom_conclusion
# Custom conclusion file(s)
@CUSTOM_CONCLUSION_FILE@
#else
!insertmacro MUI_PAGE_FINISH
#endif
```

This is just a suggestion code-wise, feel free to implement it in a different way. If you do use the `#if` conditionals, you'll need to edit the `ppd` dictionary in `winexe.py`.